doc/radosgw/account.rst

```diff
@@ -163,6 +163,46 @@ To enable a bucket quota for the account::
     radosgw-admin quota set --quota-scope=bucket --account-id={accountid} --max-objects=1000000
     radosgw-admin quota enable --quota-scope=bucket --account-id={accountid}
 
+
+Account Summary
+---------------
```

Comment:
i don't think account.rst is the right place for this. it's not about iam apis, but how to set up and use the account feature

in iam.rst, i chose not to write detailed api documentation like this. they essentially duplicate the aws docs, and we don't have a process to keep them consistent with code changes

Reply:
@cbodley : can you suggest a good place to add what fields will be part of the output of GetAccountSummary implemented in rgw